nilmtk/api.py

```diff
@@ -1,6 +1,7 @@
 from nilmtk.dataset import DataSet
 from nilmtk.metergroup import MeterGroup
 import pandas as pd
+from six import iteritems
```

Comment:
If you just remove this line, seems fine to merge it. 
There are some superfluous whitespace changes too -- don't need to change it for this but try to keep it in mind in the future.

Reply:
(we don't use/depend on `six` anymore, that's why we need to remove it)